timelock-server/build.gradle

```diff
@@ -27,8 +27,14 @@ dependencies {
     compile group: 'com.palantir.remoting1', name: 'tracing'
     compile group: 'io.atomix', name: 'atomix'
     compile group: 'io.atomix.catalyst', name: 'catalyst-netty'
-    compile group: 'io.dropwizard', name: 'dropwizard-core'
+    compile group: 'io.dropwizard', name: 'dropwizard-core', version: libVersions.timelock_dropwizard
+    compile group: 'io.dropwizard', name: 'dropwizard-http2', version: libVersions.timelock_dropwizard
+    compile group: 'io.dropwizard', name: 'dropwizard-request-logging', version: libVersions.timelock_dropwizard
     compile group: 'io.dropwizard.modules', name: 'dropwizard-java8'
+    compile group: 'org.eclipse.jetty', name: 'jetty-alpn-server'
+    compile group: 'org.eclipse.jetty.http2', name: 'http2-common'
+
+    runtime group: 'org.mortbay.jetty.alpn', name: 'jetty-alpn-agent'
```

Comment:
Yep. The ALPN stuff isn't brought in by the atlasdb-dropwizard-bundle at all (it doesn't support HTTP/2 connectors) though, so no separate config is needed here.